scripts/aws-deploy.sh

```diff
@@ -0,0 +1,288 @@
+$psed -i -e "s/REGISTRY_SSL=.*/REGISTRY_SSL=$REGISTRY_SSL/" .env
+$psed -i -e "s/REGISTRY_URL=.*/REGISTRY_URL=$REGISTRY_DOMAIN/" .env
+$psed -i -e "s/REGISTRY_USER=.*/REGISTRY_USER=$REGISTRY_USER/" .env
+
+# S3
+$psed -i -e "s/S3_ACCESS_KEY=.*/S3_ACCESS_KEY=$ACCESS_KEY_ID/" .env
+$psed -i -e "s~S3_SECRET_KEY=.*~S3_SECRET_KEY=$SECRET_ACCESS_KEY~" .env
+$psed -i -e "s/S3_BUCKET_NAME=.*/S3_BUCKET_NAME=$S3_BUCKET/" .env
+$psed -i -e "s/S3_ENDPOINT=.*/S3_ENDPOINT=$S3_ENDPOINT/" .env
+$psed -i -e "s/S3_SECURE=.*/S3_SECURE=$S3_SECURE/" .env
+
+# mail
+$psed -i -e "s/SMTP_HOST=.*/SMTP_HOST=$SMTP_HOST/" .env
+$psed -i -e "s/SMTP_PORT=.*/SMTP_PORT=$SMTP_PORT/" .env
+$psed -i -e "s/SMTP_USERNAME=.*/SMTP_USERNAME=$SMTP_USERNAME/" .env
+$psed -i -e "s/SMTP_PASSWORD=.*/SMTP_PASSWORD=$SMTP_PASSWORD/" .env
+
+# Osparc config
+$psed -i -e "s/WEBSERVER_LOGIN_REGISTRATION_CONFIRMATION_REQUIRED=.*/WEBSERVER_LOGIN_REGISTRATION_CONFIRMATION_REQUIRED=$WEBSERVER_LOGIN_REGISTRATION_CONFIRMATION_REQUIRED/" .env
+$psed -i -e "s/WEBSERVER_LOGIN_REGISTRATION_INVITATION_REQUIRED=.*/WEBSERVER_LOGIN_REGISTRATION_INVITATION_REQUIRED=$WEBSERVER_LOGIN_REGISTRATION_INVITATION_REQUIRED/" .env
+$psed -i -e "s/WEBSERVER_STUDIES_ACCESS_ENABLED=.*/WEBSERVER_STUDIES_ACCESS_ENABLED=$WEBSERVER_STUDIES_ACCESS_ENABLED/" .env
+
+# Rabbit
+$psed -i -e "s/RABBIT_PORT=.*/RABBIT_PORT=$RABBIT_PORT/" .env
+$psed -i -e "s/RABBIT_HOST=.*/RABBIT_HOST=$RABBIT_HOST/" .env
+$psed -i -e "s/RABBIT_LOG_CHANNEL=.*/RABBIT_LOG_CHANNEL=$RABBIT_LOG_CHANNEL/" .env
+$psed -i -e "s/RABBIT_PROGRESS_CHANNEL=.*/RABBIT_PROGRESS_CHANNEL=$RABBIT_PROGRESS_CHANNEL/" .env
+$psed -i -e "s/RABBIT_USER=.*/RABBIT_USER=$RABBIT_USER/" .env
+$psed -i -e "s/RABBIT_PASSWORD=.*/RRABBIT_PASSWORD=$RABBIT_PASSWORD/" .env
+
+# Reddis
+$psed -i -e "s/REDIS_HOST=.*/REDIS_HOST=$REDIS_HOST/" .env
+$psed -i -e "s/REDIS_PORT=.*/REDIS_PORT=$REDIS_PORT/" .env
+
+
+# docker-compose-simcore
+ori_compose_simcore=`cat docker-compose.deploy.yml`
+
+# Traefik does not use https
+$psed -i -e 's/traefik.http.routers.${PREFIX_STACK_NAME}_webserver.entrypoints=.*/traefik.http.routers.${PREFIX_STACK_NAME}_webserver.entrypoints=http/' docker-compose.deploy.yml
+$psed -i -e 's/traefik.http.routers.${PREFIX_STACK_NAME}_webserver.tls=.*/traefik.http.routers.${PREFIX_STACK_NAME}_webserver.tls=false/' docker-compose.deploy.yml
+
+# We don't use a auto-generated root certificate for storage
+$psed -i -e 's/\s\s\s\ssecrets:/    #secrets:/' docker-compose.deploy.yml
+$psed -i -e 's/\s\s\s\s\s\s- source: rootca.crt/      #- source: rootca.crt/' docker-compose.deploy.yml
+$psed -i -e "s~\s\s\s\s\s\s\s\starget: /usr/local/share/ca-certificates/osparc.crt~        #target: /usr/local/share/ca-certificates/osparc.crt~" docker-compose.deploy.yml
+$psed -i -e 's~\s\s\s\s\s\s- SSL_CERT_FILE=/usr/local/share/ca-certificates/osparc.crt~      #- SSL_CERT_FILE=/usr/local/share/ca-certificates/osparc.crt~' docker-compose.deploy.yml
```

Comment:
prefer composition.
you can easily create a docker-compose.aws.yml file where these changes are written instead of using sed which will fail as soon as we a small change in the syntax.

Reply:
Add that in the deployer config